### PR TITLE
Update compute plan

### PR DESCRIPTION
**Pull request by jmorel**
- New sdk method: `update_compute_plan`
- New CLI asset (`compute_plan`) for command `update`

Companion PRs:
* https://github.com/SubstraFoundation/substra-backend/pull/137
* https://github.com/SubstraFoundation/substra-tests/pull/69
* https://github.com/SubstraFoundation/substra-chaincode/pull/89

### CODE REVIEW

File: substra/cli/printers.py [hunk 1]
```diff
@@ -128,6 +128,19 @@ def get_value(self, item, expand=False):
         return f'{done_count}/{tuple_count}'
 
 
+class MappingField(Field):
+    def get_value(self, item, expand=False):
+        mapping = super().get_value(item) or {}
+        if expand:
+            value = [
+                f'{key}:{mapping[key]}'
+                for key in mapping.keys()
```
Comment (samlesu) [inline substra/cli/printers.py]:
```suggestion
                for key, v in mapping.items()
```

Minor: no need to access the value using `mapping[key]`.

File: substra/cli/printers.py [hunk 2]
```diff
@@ -128,6 +128,19 @@ def get_value(self, item, expand=False):
...
+            ]
+        else:
+            value = f'{len(mapping.keys())} values'
```
Comment (samlesu) [inline substra/cli/printers.py]: Minor: you can call `len()`  directly on the dict object: `len(mapping)`.

File: substra/cli/printers.py (hunk 1 above)
Comment (jmorel) [reply substra/cli/printers.py]: Very good point!

**Comment by jmorel** [comment]
@samlesu updated! I also updated the doc to match #112 for the new command/method.

**Pull request merged**